user-operator/src/test/java/io/strimzi/operator/user/operator/SimpleAclOperatorTest.java

```diff
@@ -62,162 +70,198 @@ public static void after() {
-        Resource res1 = new Resource(Topic$.MODULE$, "my-topic", PatternType.LITERAL);
-        Resource res2 = new Resource(Group$.MODULE$, "my-group", PatternType.LITERAL);
-        scala.collection.immutable.Set<Acl> set1 = new scala.collection.immutable.Set.Set3<>(fooAcl, barAcl, allAcl);
-        scala.collection.immutable.Set<Acl> set2 = new scala.collection.immutable.Set.Set2<>(bazAcl, anonymousAcl);
-        scala.collection.immutable.Map<Resource, scala.collection.immutable.Set<Acl>> map = new scala.collection.immutable.Map.Map2<>(res1, set1, res2, set2);
-        when(mockAuthorizer.getAcls()).thenReturn(map);
-
-        ArgumentCaptor<KafkaPrincipal> principalCaptor = ArgumentCaptor.forClass(KafkaPrincipal.class);
-        when(mockAuthorizer.getAcls(principalCaptor.capture())).thenReturn(map);
-
-        assertThat(aclOp.getUsersWithAcls(), is(new HashSet(asList("foo", "bar", "baz"))));
+        Admin mockAdminClient = mock(AdminClient.class);
+        SimpleAclOperator aclOp = new SimpleAclOperator(vertx, mockAdminClient);
+
+        ResourcePattern res1 = new ResourcePattern(ResourceType.TOPIC, "my-topic", PatternType.LITERAL);
+        ResourcePattern res2 = new ResourcePattern(ResourceType.GROUP, "my-group", PatternType.LITERAL);
+
+        KafkaPrincipal foo = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "CN=foo");
+        AclBinding fooAclBinding = new AclBinding(res1, new AccessControlEntry(foo.toString(), "*",
+                org.apache.kafka.common.acl.AclOperation.READ, AclPermissionType.ALLOW));
+        KafkaPrincipal bar = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "CN=bar");
+        AclBinding barAclBinding = new AclBinding(res1, new AccessControlEntry(bar.toString(), "*",
+                org.apache.kafka.common.acl.AclOperation.READ, AclPermissionType.ALLOW));
+        KafkaPrincipal baz = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "baz");
+        AclBinding bazAclBinding = new AclBinding(res2, new AccessControlEntry(baz.toString(), "*",
+                org.apache.kafka.common.acl.AclOperation.READ, AclPermissionType.ALLOW));
+        KafkaPrincipal all = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "*");
+        AclBinding allAclBinding = new AclBinding(res1, new AccessControlEntry(all.toString(), "*",
+                org.apache.kafka.common.acl.AclOperation.READ, AclPermissionType.ALLOW));
+        KafkaPrincipal anonymous = new KafkaPrincipal(KafkaPrincipal.USER_TYPE, "ANONYMOUS");
+        AclBinding anonymousAclBinding = new AclBinding(res2, new AccessControlEntry(anonymous.toString(), "*",
+                org.apache.kafka.common.acl.AclOperation.READ, AclPermissionType.ALLOW));
+
+        Collection<AclBinding> aclBindings =
+                asList(fooAclBinding, barAclBinding, bazAclBinding, allAclBinding, anonymousAclBinding);
+
+        try {
+            mockDescribeAcls(mockAdminClient, AclBindingFilter.ANY, aclBindings);
```

Comment:
```
assertDoesNotThrow(() ->  mockDescribeAcls(mockAdminClient, AclBindingFilter.ANY, aclBindings))
```
is a good alternative aslong as we don't need the return value to this,
also means we don't have to add to the list of handled exceptions if they change.

Similar suggestion applies to all try{} catch{} in this file

Reply:
Good suggestion! Done ;-)